client/state/stored-cards/actions.js

```diff
@@ -32,6 +35,42 @@ export const addStoredCard = ( cardData ) => ( dispatch ) => {
 		} );
 };
 
+export const editStoredCardTaxLocation = ( card, tax_postal_code, tax_country_code ) => (
+	dispatch
+) => {
+	dispatch( {
+		type: STORED_CARDS_EDIT,
+		card,
+	} );
+
+	return Promise.all(
+		card.allStoredDetailsIds.map( ( stored_details_id ) =>
```

Comment:
Where does `allStoredDetailsIds` come from? I don't see it on the `PaymentMethod` object type (which is what `card` is) and it doesn't appear to be defined in this diff.